code/integration-tests/runtime-tests/test/tests/vesting/vestingTests.ts

```diff
@@ -2,271 +2,670 @@ import { ApiPromise } from "@polkadot/api";
-    expect(Number(initialBalance2["free"])).to.be.lt(Number(finalBalance2["free"]));
-    expect(Number(currencyId)).to.be.eq(usdt);
-    expect(Number(amount)).to.be.eq(vestingPeriodCount * perPeriodAmount);
+    const txs = [
+      api.tx.sudo.sudo(
+        api.tx.vesting.vestedTransfer(walletFundSender.publicKey, walletBeneficiary.publicKey, assetId, scheduleInfo)
+      ),
+      api.tx.sudo.sudo(
+        api.tx.vesting.vestedTransfer(walletFundSender.publicKey, walletBeneficiary.publicKey, assetId, scheduleInfo)
+      ),
+      api.tx.sudo.sudo(
+        api.tx.vesting.vestedTransfer(walletFundSender.publicKey, walletBeneficiary.publicKey, assetId, scheduleInfo)
+      )
+    ];
+    const results = await sendWithBatchAndWaitForSuccess(
+      api,
+      sudoKey,
+      api.events.vesting.VestingScheduleAdded.is,
+      txs,
+      false
+    );
+
+    const vestingIds: u128[] = [];
+    for (let i = 0; i < txs.length; i++) {
+      // @ts-ignore
+      vestingIds.push(api.createType("u128", results.valueOf(i).data[3].toNumber()));
+    }
+    vestingScheduleIdCollection = vestingIds;
   });
 
-  it("beneficiary should be able to claim available funds", async function () {
-    await waitForBlocks(api, 4);
-    const asset = api.createType("u128", usdt);
-    const vestingScheduleId = api.createType("ComposableTraitsVestingVestingScheduleIdSet", "All");
+  it("#1.15 I can not make a vested transfer without sudo rights.", async function () {
+    this.timeout(2 * 60 * 1000);
+    // Parameters
+    const walletFundSender = wallet1;
+    const walletBeneficiary = wallet5;
+    const assetId = new BN(129);
+    const momentStart = 10;
+    const momentPeriod = 10;
+    const schedulePeriodCount = new BN(10);
+    const schedulePerPeriod = new BN(10).pow(new BN(12));
+    const scheduleInfo = api.createType("ComposableTraitsVestingVestingScheduleInfo", {
+      window: api.createType("ComposableTraitsVestingVestingWindow", {
+        BlockNumberBased: {
+          start: api.createType("BlockNumber", momentStart),
+          period: api.createType("BlockNumber", momentPeriod)
+        }
+      }),
+      periodCount: schedulePeriodCount,
+      perPeriod: schedulePerPeriod // 1 PICA per period
+    });
 
-    const rawInitialBalance2 = await api.query.tokens.accounts(wallet2.address, usdt);
-    const initialBalance2 = rawInitialBalance2.toJSON();
+    const verificationHandler = new vestedScheduleCreationVerifier(api);
+    verificationHandler.verificationSetup(assetId, walletFundSender.publicKey);
 
-    const {
-      data: [, assetId, , lockedAmount, claimedAmount]
-    } = await sendAndWaitForSuccess(
+    await sendAndWaitForSuccess(
+      api,
+      walletFundSender,
+      api.events.vesting.VestingScheduleAdded.is,
+      api.tx.vesting.vestedTransfer(walletFundSender.publicKey, walletBeneficiary.publicKey, assetId, scheduleInfo)
+    ).catch(exc => {
+      expect(exc.toString()).to.contain("BadOrigin");
```

Comment:
I'm not sure if this assertion will be checked if the transaction succeeds. In Jest you have `expect.assertions(n)` to make sure you don't miss any. Maybe there's something similar in Chai. Otherwise, you can change some boolean within the catch block and check its value after that.